### PR TITLE
Update NLDI landing page

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Before making a pull request
----------------------------

- [x] Update the changelog appropriately

Description
-----------
In prep for the webinar on the 25th, I want to get this updated. This is draft for review -- I'd like to have @EthanGrahn, @jkreft-usgs and @amrhoades approve before merging.

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by amrhoades** [comment]
@dblodgett-usgs - I'll review this Friday

File: content/nldi.md [hunk 1]
```diff
@@ -54,43 +55,59 @@ The public face of the NLDI is a search service that takes a watershed identifie
 How is the NLDI service structured?
 -----------------------------------------------------------------------
 
-See the swagger documentation here: [https://labs.waterdata.usgs.gov/api/nldi/swagger-ui.html](https://labs.waterdata.usgs.gov/api/nldi/swagger-ui.html)
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data` returns indexed feature sources.
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data/featureSource/featureID/navigation` returns the navigation options for that feature ID from the given feature source.
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data/featureSource/featureID/navigation/mode/dataSource` returns the data derived from a navigation with the provided mode for the requested data source.
+*   `https://labs.waterdata.usgs.gov/api/nldi/linked-data/featureSource/featureID/basin` returns a basin boundary upstream of that featureID.
```
Comment (EthanGrahn) [inline content/nldi.md:61]: I'd recommend adding an indicator as to which parts of the url are variables. (e.g. `https://labs.waterdata.usgs.gov/api/nldi/linked-data/{featureSource}/{featureID}/navigation`)

Comment (dblodgett-usgs) [reply content/nldi.md]: Just added. 
image

**Comment by dblodgett-usgs** [comment]
@jkreft-usgs -- I think this is ready to look at on labs-beta. If we merge, is there a deployer in place? I know we are working on a new one, but I'm hoping we can get this and one other change in on the old deployer...

**Comment by jkreft-usgs** [comment]
fingers crossed that the crufty old Jenkins job still works!

**Pull request merged**